cukinia

```diff
@@ -822,6 +823,47 @@ _cukinia_gpio_sysfs()
 	return 0
 }
 
+# _cukinia_knoerror: Check the kernel has booted without important errors
+# arg1: the message priority
+_cukinia_knoerror()
+{
+	local priority="$1"
+	local ret
+	local use_journalctl
+
+	if [ -x "$(which journalctl)" ]; then
+		use_journalctl=1
+	else
+		use_journalctl=0
+	fi
+
+	_cukinia_prepare "Checking that kernel has booted with no error (priority: $priority)"
+
+	if [ "$use_journalctl" -eq 1 ]; then
+		journalctl --dmesg --priority "$priority" | grep "No entries"
+		ret=$?
+	else
+		if dmesg --level "$priority"; then
```

Comment:
this one does not behave the same as journalctl. `journalctl --dmesg --priority N` will print messages with priority >=N, while `dmesg --level N` will use ==N, which means we will get different results.